jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -234,15 +238,20 @@ public boolean clearCurrentSelection( CellPreviewEvent<ProcessInstanceSummary> e
     @Override
     public void initColumns( ExtendedPagedTable<ProcessInstanceSummary> extendedPagedTable ) {
         final ColumnMeta checkColumnMeta = initChecksColumn();
-
-        actionsColumn = initActionsColumn();
+        Column<ProcessInstanceSummary, ?> actionsColumn = initActionsColumn();
+        Column<ProcessInstanceSummary, ?> errorCountColumn = initErrorCountColumn();
+        
+        ignoreSelectionColumns.add(actionsColumn);
+        ignoreSelectionColumns.add(errorCountColumn);
 
         final List<ColumnMeta<ProcessInstanceSummary>> columnMetas = new ArrayList<ColumnMeta<ProcessInstanceSummary>>();
 
         columnMetas.add(checkColumnMeta);
         columnMetas.add(new ColumnMeta<>(createTextColumn(COLUMN_PROCESS_INSTANCE_ID,
                                                           process -> String.valueOf(process.getProcessInstanceId())),
                                          constants.Id()));
+        
+        columnMetas.add(new ColumnMeta<>(errorCountColumn, constants.Errors()));
```

Comment:
No problem, I just followed the order displayed on the visual mockup. I'll change it